bookshelf/gce/deploy.sh

```diff
@@ -0,0 +1,151 @@
+# Copyright 2015 Google Inc.
+#d
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#! /bin/bash
+
+set -ex
+
+ZONE=us-central1-f
```

Comment:
yeah it's possible. I copied these scripts directly from the other getting-started repos.